controller/internal/enforcer/nfqdatapath/autoport_linux.go

```diff
@@ -0,0 +1,125 @@
+// +build linux
```

Comment:
again, file ends in `_linux.go`, doesn't need this line

Reply:
autoport.go is common to all builds, whereas autoport_linux.go and autoport_windows.go provide platform-specific impl. but this reminds me that there is no osx/darwin build for these functions, so let me add that.